src/components/sidebar/sidbar.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState} from 'react'
+import { useFilter } from "../../context/filterContext/filterContext"
+import { CategoryFilter } from './categoryFilter';
+import { PriceFilter } from './priceFilter';
+import { SortLhHl } from './sort-lh-hl';
+
+
+
+function Sidebar() {
+  const { dispatch } = useFilter();
+  const [rating, setRating] = useState(5)
+  const [toggleSideBar,setToggleSideBar]=useState(false);
+
+  const filterBar=()=>{
+    setToggleSideBar(prev=>!prev)
+  }
+
+  const btnClearClickHandler = () => {
+
+    //setting filter input checked false
+    const elements = document.getElementsByTagName("input");
```

Comment:
avoid using document.getElementsByTagName

Reply:
ok thanks fro the review